pynitrokey/cli/storage.py

```diff
@@ -49,5 +49,120 @@ def enable_update(password):
         local_print("No Nitrokey Storage device found")
 
 
+@click.command()
+@click.option(
+    "-p",
+    "--password",
+    prompt="User PIN required",
+    hide_input=True,
+    help="User password",
+)
+def open(password):
+    """Unlock the encrypted storage"""
+    try:
+        nks = NitrokeyStorage()
+        nks.connect()
```

Comment:
What do you think about moving the connection code and the error handling into a helper function to avoid repeating it in every command?